### DISCUSSION TITLE
How to manage userflow branches

### DISCUSSION THREAD

**Discussion by AbrahamBrookes** [cypress-io/cypress#19144] [Component Testing]
I have a user flow where, right at the end of the flow after the task is completed, the app gives the user four buttons to click. I need to run the entire test four times in order to test all four buttons. Is there a nicer way to do this that doesn't involve copy pasting the entire test? I guess I could make a helper function but this seems like the kind of thing that cypress might intelligently handle. Is there a time-travel feature or something like?

**Comment by bahmutov** [comment]
You could use cypress-each plug-in
